code/modules/mob/living/simple_animal/abnormality/waw/generalb.dm

```diff
@@ -0,0 +1,176 @@
+	threat_level = WAW_LEVEL
+	start_qliphoth = 1
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(0, 0, 55, 55, 60),
+						ABNORMALITY_WORK_INSIGHT = list(0, 0, 45, 45, 50),
+						ABNORMALITY_WORK_ATTACHMENT = 0,						//DO NOT FUCK THE BEEGIRL
+						ABNORMALITY_WORK_REPRESSION = list(0, 0, 40, 40, 40)
+						)
+	work_damage_amount = 10
+	work_damage_type = RED_DAMAGE
+	ego_list = list(
+		/datum/ego_datum/weapon/loyalty,
+		/datum/ego_datum/armor/loyalty
+		)
+	//She doesn't usually breach. However, when she does, she's practically an Aleph-level threat. She's also really slow, and should pack a punch.
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.3, WHITE_DAMAGE = 0.6, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 1)
+	melee_damage_lower = 40
+	melee_damage_upper = 52
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	//She has a Quad Artillery Cannon
+	var/fire_cooldown_time = 3 SECONDS	//She has 4 cannons, fires 4 times faster than the artillery bees
+	var/fire_cooldown
+	var/fireball_range = 30
+	var/volley_count
+
+/mob/living/simple_animal/hostile/abnormality/general_b/neutral_effect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(40))
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/general_b/failure_effect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(80))
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/general_b/zero_qliphoth(mob/living/carbon/human/user)
+	if(!(status_flags & GODMODE)) // If it's breaching right now
+		return	//Yeah don't increase Qliphoth
+	spawn_bees()
+	datum_reference.qliphoth_change(1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/general_b/Life()
+	. = ..()
+	if(!.) // Dead
+		return FALSE
+	if((fire_cooldown < world.time))
+		fireshell()
+
+/mob/living/simple_animal/hostile/abnormality/general_b/proc/fireshell()
+	fire_cooldown = world.time + fire_cooldown_time
+	var/list/targets = list()
+	for(var/mob/living/carbon/human/L in livinginrange(fireball_range, src))
+		if(faction_check_mob(L, FALSE))
+			continue
+		if(L.stat == DEAD)
+			continue
+		targets += L
+	new /obj/effect/beeshell(get_turf(pick(targets)))
+	volley_count+=1
+	if(volley_count>=4)
+		volley_count=0
+		fire_cooldown = world.time + fire_cooldown_time*3	//Triple cooldown every 4 shells
+
+/mob/living/simple_animal/hostile/abnormality/general_b/breach_effect()
+	icon_state = "generalbee_breach"
+	addtimer(CALLBACK(GLOBAL_PROC, .proc/show_global_blurb, 5 SECONDS, "My queen? I hear your cries...", 25))
+	SLEEP_CHECK_DEATH(80)
+	var/turf/T = pick(GLOB.department_centers)
+	forceMove(T)
+	icon_living = "general_breach"
+	icon_state = icon_living
+	var/turf/orgin = get_turf(src)
+	var/list/all_turfs = RANGE_TURFS(2, orgin)
+	for(var/turf/Y in all_turfs)
+		if(prob(60))
+			new /mob/living/simple_animal/hostile/worker_bee/soldier(Y)
+		else if(prob(20))
+			new /mob/living/simple_animal/hostile/artillery_bee(T)
```

Comment:
This spawns at T instead of Y, what is that referencing as a Turf to spawn at?

Reply:
Right, I was using T before for this and I forgot to change it